### PR TITLE
Add enterprise list runner applications download

### PR DESCRIPTION
**Pull request by gabriel-samfira**
This change adds ListRunnerApplicationDownloads() to the Enterprise service.

Fixes: #2495

Signed-off-by: Gabriel Adrian Samfira <gsamfira@cloudbasesolutions.com>

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @gabriel-samfira !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
